### DISCUSSION TITLE
CI with global tox configuration

### DISCUSSION THREAD

**Discussion by H4ppy-04** [initd1/HackAlert#45] [Ideas]
- **Regarding**: https://github.com/initd1/HackAlert/issues/29#issuecomment-1501921524
 - **Subject**: Project integration

 - I was thinking that tox might be a good idea so that we're all running the same tests with the same arguments. Regarding docker, I'll have to do more research.
 - If you'd like, I can create an issue from this discussion, create a feature branch and draft out a simple tox file if that's ok? Alternatively, looking at default CI actions (linting, coverage, etc) might also be feasible.

**Comment by initd1** [comment]
Thanks @GH-Syn. Please feel free to draft a tox file and I am happy to trial it. 

I agree that linting and coverage can be covered by CI actions. I am new to Github Actions, so if you have the know-how, please let me know. 

Regarding Docker, I will do the same of creating an issue and demonstrate how it can be used for packaging different versions into docker images which can then be used for easy distribution.
